databricks/koalas/series.py

```diff
@@ -317,8 +317,61 @@ def unique(self):
         # Pandas wants a series/array-like object
         return _col(self.to_dataframe().unique())
 
-    @derived_from(pd.Series)
     def value_counts(self, normalize=False, sort=True, ascending=False, bins=None, dropna=True):
+        """
+        Return a Series containing counts of unique values.
+        The resulting object will be in descending order so that the
+        first element is the most frequently-occurring element.
+        Excludes NA values by default.
+        Parameters
+        ----------
+        normalize : boolean, default False
+            If True then the object returned will contain the relative
+            frequencies of the unique values.
+        sort : boolean, default True
+            Sort by values.
+        ascending : boolean, default False
+            Sort in ascending order.
+        bins : Not Yet Supported
+        # TODO: Update Documentation when its supported
```

Comment:
Can we leave this todo out of the docstring?

Reply:
Actually, let's do this here. We can exclude what we don't support in this docstring, and leave a short todo comment what's excluded right above of `value_counts` definition.